parquet-avro/src/main/java/org/apache/parquet/avro/AvroReadSupport.java

```diff
@@ -112,7 +112,7 @@ public ReadContext init(Configuration configuration,
 
     if (readContext.getReadSupportMetadata().get(AVRO_READ_SCHEMA_METADATA_KEY) != null) {
```

Comment:
`readContext.getReadSupportMetadata()` can be replaced with `metadata` here as well